### PR TITLE
Feature/oauth copyfromurl

### PR DESCRIPTION
**Pull request by jaschrep-msft**
Adds bearer token arguments for copying from URL across blobs and files.

### CODE REVIEW

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/ContainersGetPropertiesHeaders.java [hunk 1]
```diff
@@ -92,6 +86,12 @@
     @JsonProperty(value = "x-ms-has-immutability-policy")
     private Boolean xMsHasImmutabilityPolicy;
 
+    /*
+     * The x-ms-version-level-worm-enabled property.
+     */
+    @JsonProperty(value = "x-ms-version-level-worm-enabled")
+    private Boolean xMsVersionLevelWormEnabled;
+
```
[the remark this answers is not in the document]
Comment (gapra-msft) [reply sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/ContainersGetPropertiesHeaders.java:94]: @jaschrep-msft Just regenerated with latest swagger and this should go away

File: sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/http/HttpAuthorization.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+// Licensed under the MIT License.
+
+package com.azure.core.experimental.http;
+
+import com.azure.core.util.CoreUtils;
+
+/**
+ * Represents the value of an HTTP Authorization header.
+ */
+public class HttpAuthorization {
```
Comment (kasobol-msft) [inline sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/http/HttpAuthorization.java:11]: I'd make this change in separate PR.

File: sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/http/HttpAuthorization.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+    /**
+     * Constructs a new HttpAuthorization instance.
+     *
+     * @param scheme Scheme component of an authorization header value.
+     * @param parameter The credentials used for the authorization header value.
+     */
+    public HttpAuthorization(String scheme, String parameter) {
+        CoreUtils.isNullOrEmpty(scheme);
+        CoreUtils.isNullOrEmpty(parameter);
```
Comment (kasobol-msft) [inline sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/http/HttpAuthorization.java:23]: this seems to return boolean but doesn't throw.

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobCopyFromUrlOptions.java [hunk 4]
```diff
@@ -118,4 +120,20 @@ public BlobCopyFromUrlOptions setDestinationRequestConditions(BlobRequestConditi
...
+
+    /**
+     * @return auth header for access to source.
+     */
+    public HttpAuthorization getSourceAuthorization() {
+        return sourceAuthorization;
+    }
+
+    /**
+     * @param sourceAuthorization auth header for access to source.
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobCopyFromUrlOptions.java:132]: we should document that we take only Bearer token at this time here and places like this.

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy [hunk 5]
```diff
@@ -2071,6 +2109,40 @@ class BlockBlobAPITest extends APISpec {
         destinationProperties.getAccessTier() == AccessTier.COOL
     }
 
+    def "Upload from URL source oauth"() {
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy:2112]: consider moving source oauth tests to separate test class.
please annotate tests (or test class if it comes to that) with `@RequiredServiceVersion`.

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy [hunk 6]
```diff
@@ -790,4 +791,13 @@ class APISpec extends StorageSpec {
             }
         }
     }
+
+    static Mono<String> getAuthToken() {
+        if (env.testMode == TestMode.PLAYBACK) {
+            return Mono.just("recordingBearerToken")
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy:797]: 1. How does this work? Does playback check if headers are matching or does it only care about URI?
2. We should make sure that source auth header is redacted in recordings. Could you check that?

File: sdk/storage/azure-storage-blob/swagger/README.md [hunk 7]
```diff
@@ -16,7 +16,7 @@ autorest --java --use:@autorest/java@4.0.x
 
 ### Code generation settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/storage-dataplane-preview/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json
+input-file: https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/44a353f3428979dc0d8891cb119d08c852722b43/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/swagger/README.md:19]: this is pointing to fork.

File: sdk/storage/azure-storage-file-share/pom.xml [hunk 8]
```diff
@@ -85,6 +90,24 @@
       <version>2.0-M4-groovy-2.5</version> <!-- {x-version-update;org.spockframework:spock-core;external_dependency} -->
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>com.azure</groupId>
+          <artifactId>azure-storage-blob</artifactId>
+          <version>12.12.0-beta.2</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+          <scope>test</scope>
+      </dependency>
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-share/pom.xml:98]: nit: indent.

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy [hunk 9]
```diff
@@ -339,6 +346,28 @@ class APISpec extends StorageSpec {
         }
     }
 
+    static BlobServiceClient getOauthBlobServiceClient() {
+        def builder = new BlobServiceClientBuilder()
+            .endpoint("https://" + env.primaryAccount.name + ".blob.core.windows.net")
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy:351]: please don't do that. `env.primaryAccount.blobEndpoint` will do it for you. Not hardcoding this format is important when we get to testing non public clouds.

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy [hunk 10]
```diff
@@ -339,6 +346,28 @@ class APISpec extends StorageSpec {
...
+        if (env.testMode != TestMode.PLAYBACK) {
+            // AZURE_TENANT_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET
+            builder.credential(new EnvironmentCredentialBuilder().build())
+        } else {
+            // Running in playback, we don't have access to the AAD environment variables, just use SharedKeyCredential.
+            builder.credential(env.primaryAccount.credential)
+        }
+        return builder.buildClient()
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy:359]: please make sure to `instrument(builder)` before building client.

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy [hunk 11]
```diff
@@ -339,6 +346,28 @@ class APISpec extends StorageSpec {
...
+    }
+
+    static Mono<String> getAuthToken() {
+        if (env.testMode == TestMode.PLAYBACK) {
+            return Mono.just("recordingBearerToken")
+        }
+        new EnvironmentCredentialBuilder().build()
+            .getToken(new TokenRequestContext().setScopes(["https://storage.azure.com/.default"]))
+            .map { it.getToken() }
+    }
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy:369]: This should go to `StorageSpec` instead of copy pasting this from blobs.

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy [hunk 12]
```diff
@@ -751,6 +753,57 @@ class FileAPITests extends APISpec {
         "ü1ü"      || _ /* Something that needs to be url encoded. */
     }
 
+    def "Copy from URL with oauth source"() {
+        given:
+        def container = instrument(new BlobServiceClientBuilder())
+            .endpoint("https://" + env.primaryAccount.name + ".blob.core.windows.net")
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy:759]: please don't hardcode endpoint pattern.

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy [hunk 13]
```diff
@@ -751,6 +753,57 @@ class FileAPITests extends APISpec {
...
+            .credential(new StorageSharedKeyCredential(env.primaryAccount.name, env.primaryAccount.key))
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy:760]: `env.primaryAccount.credential`

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy [hunk 14]
```diff
@@ -751,6 +753,57 @@ class FileAPITests extends APISpec {
...
+
+        cleanup:
+        container.delete()
+    }
+
+    def "Copy from URL with oauth source invalid credential"() {
+        given:
+        def container = instrument(new BlobServiceClientBuilder())
+            .endpoint("https://" + env.primaryAccount.name + ".blob.core.windows.net")
+            .credential(new StorageSharedKeyCredential(env.primaryAccount.name, env.primaryAccount.key))
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAPITests.groovy:787]: same here.

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy [hunk 15]
```diff
@@ -339,6 +346,28 @@ class APISpec extends StorageSpec {
         }
     }
 
+    static BlobServiceClient getOauthBlobServiceClient() {
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy:349]: is this used ?

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy [hunk 16]
```diff
@@ -2071,6 +2109,40 @@ class BlockBlobAPITest extends APISpec {
...
+        setup:
+        def sourceBlob = cc.getBlobClient(generateBlobName())
+        sourceBlob.upload(data.defaultBinaryData)
+        def oauthHeader = getAuthToken().block()
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy:2116]: I think we can move this `block` inside getAuthToken. I don't think we'll ever chain some reactive flow from this.

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy (hunk 6 above)
Comment (jaschrep-msft) [reply sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy:797]: Playback doesn't care about oauth tokens. we just need to have some sort of string for the SDK to work and we don't want to actually try and fetch a token in playback mode.

I see no bearer tokens. I'm not sure what in our infra sets this but the outcomes appear to be fine.

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobContainerItemProperties.java [hunk 17]
```diff
@@ -348,20 +348,18 @@ public BlobContainerItemProperties setRemainingRetentionDays(Integer remainingRe
     }
 
     /**
-     * Get the isImmutableStorageWithVersioningEnabled property: Indicates if version level worm is enabled on this
-     * container.
+     * Get the isVersionLevelWormEnabled property: Indicates if version level worm is enabled on this container.
      *
-     * @return the isImmutableStorageWithVersioningEnabled value.
+     * @return the isVersionLevelWormEnabled value.
```
Comment (gapra-msft) [inline sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobContainerItemProperties.java:353]: did you accidentally manually change this during the merge? Can we change it back?

Comment (gapra-msft) [reply sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobContainerItemProperties.java:353]: same for below as well

File: sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlockBlobStageBlockFromUrlOptions.java [hunk 18]
```diff
@@ -0,0 +1,127 @@
...
+     */
+    public HttpAuthorization getSourceAuthorization() {
+        return sourceAuthorization;
+    }
+
+    /**
+     * @param sourceAuthorization auth header to access source.
+     * @return The updated options.
+     */
+    public BlockBlobStageBlockFromUrlOptions setSourceAuthorization(HttpAuthorization sourceAuthorization) {
```
Comment (gapra-msft) [inline sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlockBlobStageBlockFromUrlOptions.java:123]: Document only Bearer token

File: sdk/storage/azure-storage-blob/swagger/src/main/java/com/azure/storage/blob/customization/BlobStorageCustomization.java [hunk 19]
```diff
@@ -135,7 +135,7 @@ public void customize(LibraryCustomization customization) {
 
         ClassCustomization blobContainerItemProperties = models.getClass("BlobContainerItemProperties");
         blobContainerItemProperties.getMethod("isEncryptionScopeOverridePrevented").setReturnType("boolean", "return Boolean.TRUE.equals(%s);", true);
-        blobContainerItemProperties.getMethod("setIsImmutableStorageWithVersioningEnabled").rename("setImmutableStorageWithVersioningEnabled");
+        //blobContainerItemProperties.getMethod("setIsImmutableStorageWithVersioningEnabled").rename("setImmutableStorageWithVersioningEnabled");
```
Comment (gapra-msft) [inline sdk/storage/azure-storage-blob/swagger/src/main/java/com/azure/storage/blob/customization/BlobStorageCustomization.java:138]: please undo this before merging and make sure it generates ok

File: sdk/storage/azure-storage-file-share/pom.xml [hunk 20]
```diff
@@ -85,6 +90,24 @@
...
+        <version>12.13.0-beta.1</version> <!-- {x-version-update;com.azure:azure-storage-blob;current} -->
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.3.1</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
```
Comment (gapra-msft) [inline sdk/storage/azure-storage-file-share/pom.xml:105]: Do we need 2 different dependencies on blob here?

File: sdk/storage/azure-storage-file-share/swagger/README.md [hunk 21]
```diff
@@ -16,7 +16,7 @@ autorest --java --use:@autorest/java@4.0.x
 
 ### Code generation settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/feature/storage/listFilesV2/specification/storage/data-plane/Microsoft.FileStorage/preview/2020-10-02/file.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/storage-dataplane-preview/specification/storage/data-plane/Microsoft.FileStorage/preview/2020-10-02/blob.json
```
Comment (gapra-msft) [inline sdk/storage/azure-storage-file-share/swagger/README.md:19]: why does this point to blob? does this even work?

Comment (jaschrep-msft) [reply sdk/storage/azure-storage-file-share/swagger/README.md:19]: This was probably an error when I was cleaning things up.

File: sdk/storage/azure-storage-file-share/pom.xml (hunk 20 above)
Comment (jaschrep-msft) [reply sdk/storage/azure-storage-file-share/pom.xml:105]: Intellij probably added one of these. will fix

File: sdk/storage/azure-storage-file-share/pom.xml [hunk 22]
```diff
@@ -85,6 +90,18 @@
...
+      <groupId>com.azure</groupId>
+      <artifactId>azure-storage-blob</artifactId>
+      <version>12.12.0</version> <!-- {x-version-update;com.azure:azure-storage-blob;dependency} -->
```
Comment (gapra-msft) [inline sdk/storage/azure-storage-file-share/pom.xml:102]: I feel like we should depend on latest of blob just cause that is usually what we do within the storage package. What do you think?

Comment (jaschrep-msft) [reply sdk/storage/azure-storage-file-share/pom.xml:102]: I don't think it makes a difference for this use case. We certainly can if we want to keep things pretty and updated.

**Review by gapra-msft — APPROVED** [review]

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy [hunk 23]
```diff
@@ -790,4 +791,15 @@ class APISpec extends StorageSpec {
             }
         }
     }
+
+    static String getAuthToken() {
+        if (env.testMode == TestMode.PLAYBACK) {
+            // we just need some string to satisfy SDK for playback mode. Recording framework handles this fine.
+            return Mono.just("recordingBearerToken")
```
[the remark this answers is not in the document]
Comment (jaschrep-msft) [reply sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy:798]: Wow yeah I missed that. This is wild. I'll fix.

[the remark this answers is not in the document]
Comment (kasobol-msft) [reply sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy:798]: What worries me is that CI passed with this.

[the remark this answers is not in the document]
Comment (jaschrep-msft) [reply sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy]: Debugged. Groovy auto-calls `toString()` when needed and the method was declared to return `String`. The method was returning `"MonoJust"` in playback mode. Since the test just needed some string to function in playback, everything was fine.

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/OAuthCopySourceTests.groovy [hunk 24]
```diff
@@ -0,0 +1,165 @@
...
+import com.azure.storage.blob.options.AppendBlobAppendBlockFromUrlOptions
+import com.azure.storage.blob.options.BlobUploadFromUrlOptions
+import com.azure.storage.blob.options.BlockBlobStageBlockFromUrlOptions
+import com.azure.storage.blob.options.PageBlobUploadPagesFromUrlOptions
+import com.azure.storage.blob.specialized.AppendBlobClient
+import com.azure.storage.blob.specialized.BlockBlobClient
+import com.azure.storage.blob.specialized.PageBlobClient
+import com.azure.storage.common.test.shared.extensions.RequiredServiceVersion
+
+@RequiredServiceVersion(clazz = BlobServiceVersion.class, min = "2020-10-02")
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/OAuthCopySourceTests.groovy:17]: Please check the format for `min` , see https://github.com/Azure/azure-sdk-for-java/blob/b507f4a4dc70d7439635f433520473735a8bf6cd/sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobAPITest.groovy#L234 .

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/AppendBlobAPITest.groovy [hunk 25]
```diff
@@ -578,6 +580,40 @@ class AppendBlobAPITest extends APISpec {
...
+        when:
+        bc.appendBlockFromUrlWithResponse(
+            new AppendBlobAppendBlockFromUrlOptions(sourceBlob.getBlobUrl())
+                .setSourceAuthorization(new HttpAuthorization("Bearer", oauthHeader)),
+            null, Context.NONE)
+
+        then:
+        thrown(BlobStorageException)
+    }
+
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/AppendBlobAPITest.groovy:616]: Should these go to new class ?

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy [hunk 26]
```diff
@@ -315,6 +316,43 @@ class BlockBlobAPITest extends APISpec {
...
+        when:
+        blockBlobClient.stageBlockFromUrlWithResponse(
+            new BlockBlobStageBlockFromUrlOptions(blockId, sourceBlob.getBlobUrl())
+                .setSourceAuthorization(new HttpAuthorization("Bearer", oauthHeader)),
+            null, Context.NONE)
+
+        then:
+        thrown(BlobStorageException)
+    }
+
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy:355]: how about these ?

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/AppendBlobAPITest.groovy (hunk 25 above)
Comment (jaschrep-msft) [reply sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/AppendBlobAPITest.groovy:616]: It looks like I pressed ctrl-c instead of ctrl-x. Will delete. I have explicitly deleted the recordings for these tests already.

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy (hunk 26 above)
Comment (jaschrep-msft) [reply sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlockBlobAPITest.groovy:355]: same as before

File: sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/PageBlobAPITest.groovy [hunk 27]
```diff
@@ -4,6 +4,9 @@
 package com.azure.storage.blob.specialized
 
 import com.azure.core.exception.UnexpectedLengthException
+import com.azure.core.experimental.http.HttpAuthorization;
+import com.azure.core.util.BinaryData
+import com.azure.core.util.Context
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/PageBlobAPITest.groovy:9]: nit: mind cleaning unused imports in tests after moving new tests to separate class ?

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy [hunk 28]
```diff
@@ -339,6 +346,17 @@ class APISpec extends StorageSpec {
...
+    static String getAuthToken() {
+        if (env.testMode == TestMode.PLAYBACK) {
+            // we just need some string to satisfy SDK for playback mode. Recording framework handles this fine.
+            return "recordingBearerToken"
+        }
+        return new EnvironmentCredentialBuilder().build()
+            .getToken(new TokenRequestContext().setScopes(["https://storage.azure.com/.default"]))
+            .map { it.getToken() }
+            .block()
+    }
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy:358]: can we move this and same code in blobs to StorageSpec ?

File: sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/OAuthCopySourceTests.groovy [hunk 29]
```diff
@@ -0,0 +1,75 @@
...
+import com.azure.core.util.Context
+import com.azure.storage.blob.BlobClient
+import com.azure.storage.blob.BlobContainerClient
+import com.azure.storage.blob.BlobServiceClientBuilder
+import com.azure.storage.common.test.shared.extensions.RequiredServiceVersion
+import com.azure.storage.file.share.models.ShareStorageException
+import com.azure.storage.file.share.options.ShareFileUploadRangeFromUrlOptions
+
+@RequiredServiceVersion(clazz = ShareServiceVersion.class, min = "V2020_10_02")
+class OAuthCopySourceTests extends APISpec {ShareFileClient primaryFileClient
```
Comment (kasobol-msft) [inline sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/OAuthCopySourceTests.groovy:13]: could you format this line?

**Review by kasobol-msft — APPROVED** [review]

**Review by gapra-msft — APPROVED** [review]
